msal/tests/dev apps/XForms/XForms.Android/MainActivity.cs

```diff
@@ -50,7 +46,34 @@ protected override void OnCreate(Bundle bundle)
 
             global::Xamarin.Forms.Forms.Init(this, bundle);
             LoadApplication(new App());
+
             App.UIParent = new UIParent(Xamarin.Forms.Forms.Context as Activity);
```

Comment:
@jennyf19 @henrik-me 
I'm confused about what we are trying to explain in this section, and why the code does not use the help function (I'm assuming this is IsSystemWebviewAvailable() from the beginning?

Reply:
This section describes the three ways developers can use the webview:
1. Use only system browser (this is default and is what we currently use)
2. Use only embedded Webview (noted in line 56)
3. Use helper method, noted in line 65-75

I left system browser as the default. Let me know if this is not clear or there is a different way of explaining this.

Reply:
thanks for the explanation, @jennyf19 .
A lot of decisions are still left to the developer. If we get back to *why* you did this work, I believe that this is because there are devices where a demanded system web browser (Chrome tabs) is not available? So I wonder if the library itself should not use option 3 by itself - which, I agree is a change, but probably not in the cases where this works today?, and then let the developers override this choice? Or am I missing something?

Reply:
@jmprieur - definitely don't want to add more complexity or work for the developers, but I don't think the library should decide for them to switch to embedded Webview, as this lowers the security bar, so the developer should be aware and want this functionality.
If the developer uses the helper method as I did in the sample, the user experience is seamless...embedded Webview will launch from the start. Let's discuss further.

Reply:
Current default should be embedded webview imo as that is parity with ADAL, also the use of system browser may fail on devices where this is not setup/installed correctly. IMO we should ensure that the user on the devices still will be able to use the product or are properly notified on what they need to do to solve the problem (we can show a dialog or we can fall back to using the embedded webview in those cases). We can log and send telemetry that we did this (also we can have an option for developers to not allow this behavior.

Reply:
@henrik-me System browser is more secure...Auth in embedded webview is not allowed for Google, so seems like we'd want MSAL to use the more secure model. 
The idea of the helper is that the developer can decide how the experience will be handled. Let's discuss this further next week w/ @jmprieur

Reply:
Security and whether the component is there or not... Customers will still have an issue thus the idea could be to show the user information about the missing components or allow the app developers to show the right message. Anyway, this should be aligned across libraries. 
Also can be different for Android vs. iOS as iOS always has a system browser. On iOS there is currently a debate on moving to SFAuthenticanSession. Is this planned?

Reply:
@henrik-me Moving to SFAuthenticationSession is currently not planned...I can add a task for it. I agree about having similar experiences across libraries.